inlang/source-code/website/src/services/auth/src/pages/oauth-callback.page.tsx

```diff
@@ -14,8 +14,7 @@ import { useLocalStorage } from "#src/services/local-storage/index.js"
  */
 export function Page() {
 	// ! Extremely important to wrap the get user function
```

Comment:
@janfjohannes why?

Reply:
@felixhaeberle  good catch that was left over fromthe removed telefunc impl.